docs/DeveloperGuide.adoc

```diff
@@ -235,13 +403,74 @@ image::CommitActivityDiagram.png[]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two different things.
 // end::undoredo[]
 
+=== Generating Statistics Feature
```

Comment:
For some reason this section is not shown in your netlify preview. You might want to look into it!